### DISCUSSION TITLE
Orca slicer won't open after fresh install

### DISCUSSION THREAD

**Discussion by bigdon26** [OrcaSlicer/OrcaSlicer#4662] [Q&A]
Orca slicer unhandled unknow exception: terminating the application

I had version 1.9 from Elegoo working, then went to 1.9.1 then to 2 beta.  I then formatted my hard drive & reloaded Windows 11 Pro now none of them work after installing them?  I get the above message after I install them and try opening it.  I love the program but a newbie to all this and could use some help?  I read somewhere on the internet someone thought it might be related to the Nvidia drivers?  Please help?  My system is a Intel core I9, 64GB ram, Nvidia 4070 TI video card,  SSD m.2 drives.

Any help would be greatly appreciated!

**Comment by Narik1697** [comment]
Hello, I have the same situation on Windows 10, same error. i9-13900kf, rtx 4070ti
![image](https://github.com/SoftFever/OrcaSlicer/assets/82543219/76158053-83da-43d7-8fb2-3528180bbbe5)
